src/modules/views/Folder/virtualized/helpers.js

```diff
@@ -13,6 +13,7 @@ export const makeRows = ({ queryResults, IsAddingFolder, syncingFakeFile }) => {
     rows.push(syncingFakeFile)
   }
 
+  // TODO: we should not modify io.cozy.files for frontend purpose
```

Comment:
Where did it happen?

Reply:
https://github.com/cozy/cozy-drive/pull/3442